src/style.css

```diff
@@ -1 +1,12 @@
-/* styles go here */
+.form-field {
+  display: block;
+  margin: 10px 0;
```

Comment:
you need bottom margins only, top margins make the logic fail
```suggestion
  margin-bottom: 10px;
```
![Screenshot 2022-11-21 at 13 27 29](https://user-images.githubusercontent.com/75496546/203039361-9d573a60-3443-4b7b-be54-f42cb27af31e.png)